account_payment_method_or_mode_fs_storage/tests/test_account_payment_method_or_mode_fs_storage.py

```diff
@@ -127,3 +133,69 @@ def test_check_use_on_payment_method(self):
 
         self.assertFalse(self.payment_method.storage)
         self.assertFalse(self.fs_storage.use_on_payment_method)
+
+    def test_payment_mode_fs_storage(self):
+        self.company.update(
```

Comment:
I think that you need to adapt the test to change the 'source' of the fs_storage